exercises/binary_converter.cpp

```diff
@@ -5,3 +5,33 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+#include <iostream>
+
+std::string convert(int n){
+      std::string res = "";
+      while(n != 0){
+         if(n % 2 == 0)
+           res += "0";
+         else 
+           res += "1";
```

Comment:
per evitare di scrivere res += due volte, puoi utilizzare la condizione ternaria, potresti anche mettere 0-1 in una variabile separata, esempio
```c++
char r = n % 2 == 0 ? "0" : "1";
res += r;
```

oppure eviti di creare `char r` e metti la condizione ternaria dopo il +=